lenskit-cli/src/main/java/org/lenskit/cli/commands/Simulate.java

```diff
@@ -0,0 +1,133 @@
+ */
+@AutoService(Command.class)
+public class Simulate implements Command {
+    private final Logger logger = LoggerFactory.getLogger(Simulate.class);
+
+    public void configureArguments(ArgumentParser parser) {
+        parser.description("Simulates Temporal Evaluator and write item's predicted score to file.");
+        ScriptEnvironment.configureArguments(parser);
+        parser.addArgument("-i", "--input-file")
+              .type(File.class)
+              .metavar("FILE")
+              .setDefault("ratings.pack")
+              .help("Packed Rating File");
+        parser.addArgument("-o", "--output-file")
+              .type(File.class)
+              .metavar("FILE")
+              .setDefault("prediction.csv")
+              .help("write predicted score to FILE");
+        parser.addArgument("-r", "--rebuild-period")
+              .type(Long.class)
+              .metavar("REBUILD PERIOD IN SECONDS")
+              .help("Rebuild Period for next build");
+        parser.addArgument("config")
+              .type(File.class)
+              .nargs("+")
+              .metavar("CONFIG")
+              .help("load algorithm configuration from CONFIG");
+    }
+
+    @Override
+    public String getName() {
+        return "simulate-temporal-evaluator";
+    }
+
+    @Override
+    public String getHelp() {
+        return "simulate temporal evaluator";
+    }
+
+    @Override
+    public void execute(Namespace opts) throws IOException, RecommenderBuildException {
+
+        Context ctx = new Context(opts);
+        TemporalEvaluator tempEval = new TemporalEvaluator();
+
+        Stopwatch timer = Stopwatch.createStarted();
+
+        Long rebuildPeriod = ctx.getRebuildPeriod();
+        tempEval.setDataSource(ctx.getInputFile());
+        tempEval.setPredictOutputFile(ctx.getOutputFile());
+        tempEval.setRebuildPeriod(rebuildPeriod);
+
+        //TODO change it for only one config file
+        for (LenskitConfiguration config : ctx.environment.loadConfigurations(ctx.getConfigFiles())) {
+            tempEval.setAlgorithm(config.toString(), config);
+        }
+
+        tempEval.execute();
+        timer.stop();
+        logger.info("evaluator executed  in {}", timer);
+        logger.info("written predicted score to {}", ctx.getOutputFile());
+        if (rebuildPeriod != null) {
+            logger.info("Rebuild period set to {}", ctx.getRebuildPeriod());
+        }
+    }
+
+    private static class Context {
+        private final Namespace options;
+        private final ScriptEnvironment environment;
+
+        public Context(Namespace opts) {
+            options = opts;
+            environment = new ScriptEnvironment(opts);
+        }
+
+        public File getInputFile() {
+            return options.get("input_file");
+        }
+
+        public File getOutputFile() {
+            return options.get("output_file");
+        }
+
+        public List<File> getConfigFiles() {
```

Comment:
If you change this to `File getConfigFile()`, then we'll just have 1 config file.